src/worker/helpers.go

```diff
@@ -193,7 +193,12 @@ func stopOL(_ string) error {
 	}
 	fmt.Printf("Send SIGINT and wait for worker to exit cleanly.\n")
 	if err := p.Signal(syscall.SIGINT); err != nil {
-		return fmt.Errorf("Failed to send SIGINT to PID %d (%s).  May require manual cleanup.\n", pid, err.Error())
+		// Failed to send SIGINT to PID  (os: process already finished).  May require manual cleanup will happen when system force shuts down.
+		if err.Error() == "os: process already finished" {
```

Comment:
do we need another case here to detect a dirty shutdown?  Seems like a few lines earlier we can already identify the case that worker.pid exists but os.FindProcess can't find it.  Seems like we ought to handle the dirty shutdown there instead of returning an error.

In general, I'm not sure whether string representations of errors are guaranteed to never change, so I wouldn't want to depend on that.